### ISSUE TITLE
Model attribute maps are wrong for property 'hasMore'

### ISSUE THREAD

**Issue by ianmiddelkamp** [mycartracks/mycartracks#2]
protected static $attributeMap = [
        'object' => 'object',
        'url' => 'url',
        'hasMore' => 'has_more',
        'data' => 'data'
    ];

should be 

 protected static $attributeMap = [
        'object' => 'object',
        'url' => 'url',
        'hasMore' => 'hasMore',
        'data' => 'data'
    ];

I have confirmed this invalid for TracksModel, and TrackpointsModel. Probably the other models are wrong as well.

**Comment by ianmiddelkamp** [comment]
Yes that is the issue. I am unfamiliar with swagger, or how it works. But I understand that you are using a generated SDK, and that it is provided as a head-start to the development process. It is definitely saving me time as I don't have to write the curl code. What i am finding is that during the deserialization process the 'hasMore' is getting lost. What is returned from the http request has valid data and then the hasMore boolean is missing after deserialization. The fix for this is to change the value for the key in the attribute map as I have had to do. My aim was to provide some information about trouble I had with the SDK so that others could use it as well. Perhaps I didn't get the settings right when I generated the SDK? I can also understand if tweaking the generated SDK to match the output of your API is not a huge priority, so I don't necessarily need to add to this forum. 

Thank you for your response. :)
